src/base/utils/io.cpp

```diff
@@ -89,7 +89,7 @@ nonstd::expected<QByteArray, Utils::IO::ReadError> Utils::IO::readFile(const Pat
         return nonstd::make_unexpected(ReadError {ReadError::ExceedSize, message});
     }
 
-    QByteArray ret {fileSize, Qt::Uninitialized};
+    QByteArray ret {static_cast<qsizetype>(fileSize), Qt::Uninitialized};
```

Comment:
Of course, the problem is unlikely, but still... Don't you need to check if the value fits in the target range? E.g. in case when QByteArray size is limited by max 32-bit integer.